test/unit-test/test_btc_sign.c

```diff
@@ -417,7 +417,8 @@ static bool _stream_prevtx(
 static void _sign(const _modification_t* mod)
 {
     // Need keystore to derive change and input scripts
-    mock_state(mod->seeded ? _mock_seed : NULL, mod->seeded ? _mock_bip39_seed : NULL);
+    mock_state(
+        mod->seeded ? _mock_seed : NULL, sizeof(_mock_seed), mod->seeded ? _mock_bip39_seed : NULL);
```

Comment:
maybe a more readable version would be something like this:

```c
mock_state(
    mod->seeded ? _mock_seed : NULL,
    sizeof(_mock_seed),
    mod->seeded ? _mock_bip39_seed : NULL)
```

so it visually separates the args - was already harder than it could be to read before imho

Reply:
clang-format decided how to format. disabling clang-format for these lines is not worth it imo.